<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgAddressEdit</class>
 <widget class="QDialog" name="dlgAddressEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>607</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_8">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="lblPatient">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Owner :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cmbPatient"/>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="lblName">
         <property name="text">
          <string>Name :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ledName"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="lblCountry">
         <property name="text">
          <string>Country :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ledCountry"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="lblRegion">
         <property name="text">
          <string>Region :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cmbRegion">
         <property name="editable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="lblZip">
         <property name="text">
          <string>Zip :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ledZip">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="maxLength">
          <number>8</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblCity">
         <property name="text">
          <string>City: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ledCity">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbCitySearch">
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>?</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLineEdit" name="ledStreet"/>
       </item>
       <item>
        <widget class="QComboBox" name="cmbStreet"/>
       </item>
       <item>
        <widget class="QLineEdit" name="ledStreetNumber">
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="lblFloor">
         <property name="text">
          <string>floor</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ledFloor">
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblDoor">
         <property name="text">
          <string>door</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ledDoor">
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="pbSave">
       <property name="text">
        <string>Ok</string>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pbSave</sender>
   <signal>clicked()</signal>
   <receiver>dlgAddressEdit</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>558</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>305</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbCancel</sender>
   <signal>clicked()</signal>
   <receiver>dlgAddressEdit</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>558</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>305</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
